documentation/Color system.md

```diff
@@ -277,7 +277,7 @@ Used to decorate elements where color does convey a specific meaning in componen
 | `--p-override-one`                        | `1`                                                                                                          |
 | `--p-override-visible`                    | `visible`                                                                                                    |
 | `--p-override-zero`                       | `0`                                                                                                          |
-| `--p-override-loading-z-index`            | `514`                                                                                                        |
+| `--p-override-loading-zindex`             | `514`                                                                                                        |
```

Comment:
This should match css `z-index`

Reply:
I think it's from the [new logic to group capital letters together](https://github.com/Shopify/polaris-react/pull/4683/commits/6aca9e0b55c8902ca24d5705cfab854342cd47fa).

Reply:
[z-index token reference](https://github.com/Shopify/polaris-react/blob/6117c999272f562141de6556c9a3cf2f3343a01a/src/utilities/theme/tokens.ts#L32)

You're right @lgriffee The new regex no longer inserts a hyphen before each capital letter. Wonder if that update was intentional.

Reply:
My bad. I added it for consistency then realised it caused a regression. Should be fine now!